src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Baselines/TestWithMoreThan6Params.generated.txt

```diff
@@ -88,7 +90,9 @@ namespace Microsoft.Extensions.Logging.Generators.Tests.TestClasses
         }
     }
 }
+/// <summary> The implementation of this type is subject to change without further notice, do not use it directly in your code. </summary>
```

Comment:
you'll need to update the baselines with the new wording

Reply:
Sure right, it's Friday evening and I'm beginning to be a bit slow